app/frontend/controllers/conditional_question_controller.js

```diff
@@ -1,36 +1,36 @@
-import { Controller } from "@hotwired/stimulus"
+import { Controller } from '@hotwired/stimulus'
 
 export default class extends Controller {
-  initialize() {
+  initialize () {
     this.displayConditional()
   }
 
-  displayConditional() {
-    if(this.element.checked) {
-      let selectedValue = this.element.value
-      let conditional_for = JSON.parse(this.element.dataset.info)
-      Object.entries(conditional_for).map(([targetQuestion, conditions]) => {
-          if(conditions.map(String).includes(String(selectedValue))) {
+  displayConditional () {
+    if (this.element.checked) {
+      const selectedValue = this.element.value
+      const conditionalFor = JSON.parse(this.element.dataset.info)
+
+      Object.entries(conditionalFor).forEach(([targetQuestion, conditions]) => {
+        if (!conditions.map(String).includes(String(selectedValue))) {
```

Comment:
Worth taking a note of this change. Changed the if else logic to prevent and empty block, and now using `forEach` instead of `map` because we’re no modifying the entries.